### ISSUE TITLE
British campaign: Salerno Scenario Issue

### ISSUE THREAD

**Issue by Cjh1953** [nicupavel/openpanzer#230]
cannot achieve victory due to missing objective. Have captured all Victory hexes but cannot progress to next scenario. Where’s the missing Victory objective!?

**Comment by salexgit** [comment]
Hi

I checked the scenario. Are you sure there is a problem? There are 11 objectives and 11 victory hexes on the map. Did you capture all of them? If so, then can you please attach here a saved game before you capture the last hex? Maybe there is a problem with the game not moving to the next scenario, not with this particular scenario.

Regards, Alex

**Comment by salexgit** [comment]
And I suppose that issue #229 is a duplicate and can be closed, right?

**Comment by Cjh1953** [comment]
Yes, sorry about that.

 

Chris

647.885.2292

**Comment by Cjh1953** [comment]
Hi Alex: Here's the game. I counted and the British possess all 11 victory hexes but the game will not shift to the next scenario. 

 

I am playing on my iPad but I couldn't attach the file on that device so I saved to the cloud, downloaded to my desktop and saved it to my drive. Now I am sending it via Outlook.

 

All of which to say that it may be an iPad related issue. 

 

Chris

647.885.2292

**Comment by salexgit** [comment]
Hi

I cannot see the attached saved game. Can you please upload it to this discussion thread through the GitHub interface?

**Comment by Cjh1953** [comment]
I don't know how. It is saved to the cloud though. Can you load it from there. 

 

189a 9509 2ea0 889e 427d 1867 57f4 df28

 

I just loaded it again and all 11 victory hexes area occupied by British or US units and it still just goes to the next turn, not the next scenario.

 

 

Chris

647.885.2292

**Comment by salexgit** [comment]
Hi

Yes, it is a bit weird. You have captured all the 11 objectives but the game still thinks that one objective has to be conquered.
Do you have a saved game before you capture the last objective? 

Regards, Alex

**Comment by Cjh1953** [comment]
Hi Alex

The short answer is no. The iPad I was playing on is in the shop and it’s even money the files will survive. If they can be restored I should have an earlier turn and can save it to the cloud and send it. It may take a few days though. 

I appreciate your help with this because otherwise the campaign is stalled 


Chris

**Comment by salexgit** [comment]
Hi

The saved game before taking the last victory hex is important because it will allow us to understand what goes wrong. We will take a look at the saved game you provided, but it could be the case that it does not provide much insight on why exactly this problem has occurred.

**Comment by Cjh1953** [comment]
Found it - hidden victory hex at 34,18. I didn’t have the precursor Salerno turn to the game I shared so I had to re-fight the entire campaign back to Salerno. 

I noticed an anomalous capture by the Germans where there didn’t seem to be anything to capture on Turn 1. Just saw it out of the corner of my eye and wasn’t looking for it since the Brits hadn’t even got ashore. Rebooted the Salerno scenario I was stuck at that i shared and ran the Stuart around that patch of the board till I found it. So, there are 12 victory hexes!

Chris
647.885.2292
